### ISSUE TITLE
Auto-remove docker containers after run

### ISSUE THREAD

**Issue by RixzZ** [Glavin001/atom-beautify#2204]
# Description

I'm using atom-beautify with beautysh on docker. Every time it executes, a new container is created and not autoremoved. The same container is not being used anymore, so after a few saves I end up with the same number of beautysh containers that times I saved a file.

# Steps to Reproduce

1. Add code to Atom editor (bash in my case)
2. Run command `Atom Beautify: Beautify Editor`
3. A new container is created in Docker every time but they don't remove itself after executing.

Docker have a flag '--rm' you can specify when running the container so it automatically remove itself when the task finishes. Can that be

# Checklist

I have:
- [X] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [X] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [X] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [ ] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ ] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by Glavin001** [comment]
I think the bug is in https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/executable.coffee#L451-L460 

Looks like other users have experienced this, too: https://twitter.com/lambdafu/status/987116195538141184

![image](https://user-images.githubusercontent.com/1885333/46240002-53d82980-c377-11e8-9ca8-751680e946b7.png)

**Comment by Glavin001** [comment]
I can reproduce:

```
❯ docker ps --all
CONTAINER ID        IMAGE                                                 COMMAND                  CREATED                  STATUS                      PORTS               NAMES
f64baf9722a0        unibeautify/elm-format                                "elm-format --yes in…"   Less than a second ago   Exited (0) 27 seconds ago                       elated_mccarthy
265d8f5a1ad4        unibeautify/elm-format                                "elm-format --help"      Less than a second ago   Exited (0) 28 seconds ago                       epic_swirles
```

**Comment by Glavin001** [comment]
@RixzZ You can use the following to clean up:

```bash
docker ps --all | grep "unibeautify/" | awk '{ print $1 }' | xargs docker rm
```

**Comment by Glavin001** [comment]
Published fix to v0.33.4
